File: OrganMatcherJPA/src/main/java/com/skilldistillery/organmatcher/entities/Address.java
package com.skilldistillery.organmatcher.entities;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Address {
	@Id
	@GeneratedValue( strategy = GenerationType.IDENTITY )
	private int id;
	private String street1;
	private String street2;
	private String city;
	private String state;
	private String zip;

}

File: organizer/src/app/models/address.ts
export class Address {

  id: number;
  street1: String;
  street2: String;
  city: String;
  state: String;
  zip: String;


 constructor(
  id?: number,
  street1?: String,
  street2?: String,
  city?: String,
  state?: String,
  zip?: String){

    this.id= id;
    this.street1= street1;
    this.street2 =street2;
    this.city= city;
    this.state=  state;
    this.zip= zip;
 }
}
